.github/dependabot.yml

```diff
@@ -2,22 +2,29 @@ version: 2
 updates:
   - package-ecosystem: github-actions
     directory: /
-    open-pull-requests-limit: 999
-    rebase-strategy: disabled
     schedule:
-      interval: daily
-      time: "00:00"
+      interval: weekly
+      day: "thursday"
```

Comment:
Wouldn't Sunday or Monday be better? Thursday seems like it will quickly not get merged the same week, and left over the weekend.

Reply:
@rmartinc @stianst thank you for the feedback, it does makes sense, updated.